tests/test_utils.py

```diff
@@ -107,3 +110,19 @@ def dummy_callback(locals_, _globals):
 
     episode_rewards, _ = evaluate_policy(model, model.get_env(), n_eval_episodes, return_episode_rewards=True)
     assert len(episode_rewards) == n_eval_episodes
+
+
+def test_VecNoise():
```

Comment:
```suggestion
def test_vec_noise():
```
to be consistent with the rest of the names